### ISSUE TITLE
Egg Moves

### ISSUE THREAD

**Issue by akaWaster** [PokeAPI/pokeapi#483]
How to get Egg Moves of pokemon?

Searched in API Documentation, but didnt find any information

**Comment by Naramsim** [comment]
Hi!

You can get the list of moves a Pokemon can have at the `/pokemon/<id>` endpoint. For Charmander: https://pokeapi.co/api/v2/pokemon/4

Then use the `moves` property. There there will be a long list of moves. You need to look for moves that have the `move_learn_method` ~equal to `egg`. For example, Charmander can learn sword-dance by egg (by hatching) on Pokemon Platinum, DiamondPearl, FireRedLeafGreen, Emerald, RubySapphire, HeartGoldSoulSilver.

Basically a pokemon can learn a move in different ways: https://pokeapi.co/api/v2/move-learn-method/ The method you want to use to filter is `egg` and has id `2`.

**Comment by akaWaster** [comment]
Thank you very much!

пт, 10 апр. 2020 г. в 17:21, Alessandro Pezzè <notifications@github.com>:
